### PR TITLE
Docs updates with ray

### PR DESCRIPTION
**Pull request by ChengyuanZhao**
updates and changes to docs from discussion with Ray

### CODE REVIEW

File: docs/src/main/asciidoc/config.adoc [hunk 1]
```diff
@@ -137,3 +134,8 @@ $ gcloud beta runtime-config configs variables set \
 ....
 $ curl -XPOST http://myapp.host.com/actuator/refresh
 ....
+
+=== Sample
+
+A https://github.com/spring-cloud/spring-cloud-gcp/tree/master/spring-cloud-gcp-samples/spring-cloud-gcp-config-sample[sample application]
+and a https://codelabs.developers.google.com/codelabs/cloud-spring-runtime-config/index.html[codelab] are available.
```
Comment (artembilan) [inline docs/src/main/asciidoc/config.adoc]: Must be on a single line: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

File: docs/src/main/asciidoc/datastore.adoc [hunk 2]
```diff
@@ -374,19 +372,61 @@ There are three ways to represent relationships between entities that are descri
...
 Fields whose types are also annotated with `@Entity` are converted to `EntityValue` and stored inside the parent entity.
+
+Here is an example of Cloud Datastore entity containing an embedded entity in JSON:
+
+[source, json]
+----
+{
+  "name" : "Alexander",
+  "age" : 50,
+  "children" : [ {
```
Comment (meltsufin) [inline docs/src/main/asciidoc/datastore.adoc]: I think a simpler example would be where it's not a collection, but just an embedded singular entity.

File: docs/src/main/asciidoc/datastore.adoc [hunk 3]
```diff
@@ -374,19 +372,61 @@ There are three ways to represent relationships between entities that are descri
...
+
 Multiple levels of embedded entities are supported.
 
 NOTE: Embedded entities don't need to have `@Id` field, it is only required for top level entities.
 
 Example:
 
-We would like to store a binary tree in Cloud Datastore.
+Entities can hold embedded entities that are their own type.
+We can store a trees in Cloud Datastore using this feature:
```
Comment (meltsufin) [inline docs/src/main/asciidoc/datastore.adoc]:
```suggestion
We can store trees in Cloud Datastore using this feature:
```

File: docs/src/main/asciidoc/datastore.adoc [hunk 4]
```diff
@@ -447,12 +487,17 @@ Collections of embedded objects can also be stored as the values of embedded Map
 
 Parent-child relationships are supported via the `@Descendants` annotation.
 
+Unlike embedded children, descendants are fully-formed entities residing in their own kinds.
+The parent entity does not have an extra field to hold the descendant entities.
+Instead, the relationship is captured in the descendants' keys, which refer to their parent entities:
```
Comment (meltsufin) [inline docs/src/main/asciidoc/datastore.adoc]: Can we have a Datastore JSON example here too?

File: docs/src/main/asciidoc/sql.adoc [hunk 5]
```diff
@@ -160,3 +126,12 @@ To fix this, re-declare the dependency in its correct version. For example, in M
   <version>42.1.1</version>
 </dependency>
 ----
+
+
+=== Sample
```
Comment (meltsufin) [inline docs/src/main/asciidoc/sql.adoc]:
```suggestion
=== Samples
```

Also, across the board,  I think I would prefer "Code Sample".

**Comment by ChengyuanZhao** [comment]
@meltsufin changes made , PTAL

File: docs/src/main/asciidoc/datastore.adoc [hunk 6]
```diff
@@ -529,6 +527,26 @@ The GQL key-literal representation for the parent `ShoppingOrder` would be:
 Key(orders, '12345')
 ----
 
+The Cloud Datastore entities exist separately in their own kinds:
+
+
+----
+{
```
Comment (meltsufin) [inline docs/src/main/asciidoc/datastore.adoc]: Can you indicate that this is the order and that the two below are purchased_item?

Comment (ChengyuanZhao) [reply docs/src/main/asciidoc/datastore.adoc]: Done, PTAL

**Pull request merged**